pkg/services/alerting/notifiers/pushover.go

```diff
@@ -193,7 +193,8 @@ func NewPushoverNotifier(model *models.AlertNotification) (alerting.Notifier, er
 	userKey := model.DecryptedValue("userKey", model.Settings.Get("userKey").MustString())
 	APIToken := model.DecryptedValue("apiToken", model.Settings.Get("apiToken").MustString())
 	device := model.Settings.Get("device").MustString()
-	priority, _ := strconv.Atoi(model.Settings.Get("priority").MustString())
+	alertingPriority, _ := strconv.Atoi(model.Settings.Get("priority").MustString())
```

Comment:
I think you could make the field name `alertingPriority`, and use this opportunity to make the `sound` field consistent too IE `sound` -> `alertingSound`

Reply:
Thanks for the review @wbrowne 😄 ! This seems to break my existing config as I gather it's expecting these values straight from the database (Unless there's a migration pattern I'm unaware of)?

Reply:
```diff
diff --git a/pkg/services/alerting/notifiers/pushover.go b/pkg/services/alerting/notifiers/pushover.go
index 59f225b568..46267de717 100644
--- a/pkg/services/alerting/notifiers/pushover.go
+++ b/pkg/services/alerting/notifiers/pushover.go
@@ -150,7 +150,7 @@ func init() {
 				Label:         "Alerting priority",
 				Element:       alerting.ElementTypeSelect,
 				SelectOptions: priorityOptions,
-				PropertyName:  "priority",
+				PropertyName:  "alertingPriority",
 			},
 			{
 				Label:         "OK priority",
@@ -176,7 +176,7 @@ func init() {
 				Label:         "Alerting sound",
 				Element:       alerting.ElementTypeSelect,
 				SelectOptions: soundOptions,
-				PropertyName:  "sound",
+				PropertyName:  "alertingSound",
 			},
 			{
 				Label:         "OK sound",
@@ -193,11 +193,11 @@ func NewPushoverNotifier(model *models.AlertNotification) (alerting.Notifier, er
 	userKey := model.DecryptedValue("userKey", model.Settings.Get("userKey").MustString())
 	APIToken := model.DecryptedValue("apiToken", model.Settings.Get("apiToken").MustString())
 	device := model.Settings.Get("device").MustString()
-	alertingPriority, _ := strconv.Atoi(model.Settings.Get("priority").MustString())
+	alertingPriority, _ := strconv.Atoi(model.Settings.Get("alertingPriority").MustString())
 	okPriority, _ := strconv.Atoi(model.Settings.Get("okPriority").MustString())
 	retry, _ := strconv.Atoi(model.Settings.Get("retry").MustString())
 	expire, _ := strconv.Atoi(model.Settings.Get("expire").MustString())
-	alertingSound := model.Settings.Get("sound").MustString()
+	alertingSound := model.Settings.Get("alertingSound").MustString()
 	okSound := model.Settings.Get("okSound").MustString()
 	uploadImage := model.Settings.Get("uploadImage").MustBool(true)
```

Reply:
Actually yeah in retrospect, it's not worth the trouble right now. Apologies!

Reply:
No problems! Thanks for confirming 😃